File: app/src/main/java/ar/com/wolox/android/example/ui/login/HomeActivity.java
package ar.com.wolox.android.example.ui.login;

import android.os.Bundle;

import ar.com.wolox.android.R;
import ar.com.wolox.wolmo.core.activity.WolmoActivity;

/**
 * A simple {@link WolmoActivity} subclass.
 */
public class HomeActivity extends WolmoActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_home);
    }

    @Override
    protected int layout() {
        return R.layout.activity_home;
    }

    @Override
    protected void init() {

    }
}

File: app/src/main/java/ar/com/wolox/android/example/ui/home/HomeActivity.kt
package ar.com.wolox.android.example.ui.home

import ar.com.wolox.android.R
import ar.com.wolox.wolmo.core.activity.WolmoActivity
import kotlinx.android.synthetic.main.fragment_home.*

/**
 * A simple {@link WolmoActivity} subclass.
 */
class HomeActivity : WolmoActivity() {

    override fun layout(): Int = R.layout.activity_base

    override fun init() {
        replaceFragment(R.id.vActivityBaseContent, HomeFragment())
    }
}